Cabal/Distribution/Types/GenericPackageDescription.hs

```diff
@@ -61,6 +65,9 @@ data Flag = MkFlag
 
 instance Binary Flag
 
+emptyFlag :: FlagName -> Flag
```

Comment:
OK. Could we maybe add Haddock like `-- | A 'Flag' initialized with default parameters (enabled by default, not manual, empty description)`?